specifyweb/frontend/js_src/tailwind.config.ts

```diff
@@ -3,15 +3,21 @@
  * https://github.com/tailwindlabs/tailwindcss/blob/master/stubs/defaultConfig.stub.js
  */
 
+import type {Config} from 'tailwindcss';
+import forms from '@tailwindcss/forms';
+
 /*
+ * REFACTOR: https://tailwindcss.com/blog/tailwindcss-v3-3#css-variables-without-the-var
+ * REFACTOR: https://tailwindcss.com/blog/tailwindcss-v3-4#style-children-with-the-variant
+ * REFACTOR: https://tailwindcss.com/blog/tailwindcss-v3-4#subgrid-support
+ * REFACTOR: https://tailwindcss.com/blog/tailwindcss-v3-4#extended-min-width-max-width-and-min-height-scales
```

Comment:
nice improvements, but didn't want to introduce any bugs in this PR, so will delay refactor to some future point